tutorials/containers/flux-sched-container.rst

```diff
@@ -0,0 +1,232 @@
+        ƒq3d755R fluxuser sleep       R      1      1   2.811s 848fc387afd7
+        ƒpcByPgK fluxuser sleep       R      1      1   3.805s 848fc387afd7
+
+But wait, what happened to our first two jobs? Add an ``-a`` for "all" to see all the jobs.
+
+
+.. code-block:: shell
+
+    $ flux jobs -a
+           JOBID USER     NAME       ST NTASKS NNODES     TIME INFO
+        ƒq3d755R fluxuser sleep       R      1      1   39.98s 848fc387afd7
+        ƒpcByPgK fluxuser sleep       R      1      1   40.97s 848fc387afd7
+        ƒDi2hxdm fluxuser hostname   CD      1      1   0.019s 848fc387afd7
+        ƒBmTaVZ9 fluxuser hostname   CD      1      1   0.025s 848fc387afd7
+
+.. note::
+
+   See if you can figure out how to list jobs by a particular status, e.g., ``R`` in the output 
+   above means "running." Try ``flux jobs --help``.
+
+
+Did you figure it out? It would be ``flux jobs --filter=RUNNING``. What if you were running a long 
+process, and you wanted to check on output? Let's do that. Here is script to loop, print, and sleep.
+
+.. code-block:: bash
+
+    #!/bin/bash
+    # Save this as loop.sh
+    for i in {0..10}; do
+        echo "Hello I am loop iteration $i."
+        sleep ${i}
+    done
+
+Now make the script executable, and submit the job with flux.
+
+.. code-block:: shell
+
+    $ chmod +x ./loop.sh
+    $ flux submit ./loop.sh
+
+
+To see output (and wait until completion) use ``flux job attach``:
+
+
+.. code-block:: shell
+
+    $ flux job attach ƒ4evXWb9Z
+    Hello I am loop iteration 0.
+    Hello I am loop iteration 1.
+    Hello I am loop iteration 2.
+    Hello I am loop iteration 3.
+    Hello I am loop iteration 4.
+    Hello I am loop iteration 5.
+
+See :core:man1:`flux-job` for more information on ``flux job attach``.
+
+
+------------
+Viewing Jobs
+------------
+
+Aside from listing jobs with ``flux jobs`` there are other ways to get metadata about jobs.
+For your running jobs, you can use ``flux pstree`` to see exactly that - a tree of processes.
+Let's say we run another sleep job:
+
+
+.. code-block:: shell
+
+    $ flux submit sleep 350
+    ƒ744GwLs
+
+The tree will show us that one sleep!
+
+
+.. code-block:: shell
+
+    $ flux pstree
+    flux
+    └── sleep
+
+Submit the same command a few more times? We see that reflected in the tree!
+
+.. code-block:: shell
+
+    $ flux submit sleep 350
+    ƒAivupEb
+    $ flux submit sleep 350
+    ƒB621bj5
+
+    $ flux pstree
+    flux
+    └── 3*[sleep]
+
+Another way to inspect a job is to look at stats, and pipe the JSON output into jq to make it pretty!
+
+
+.. code-block:: shell
+
+    $ flux job stats ƒB621bj5  | jq
+    {
+      "job_states": {
+        "depend": 0,
+        "priority": 0,
+        "sched": 0,
+        "run": 3,
+        "cleanup": 0,
+        "inactive": 0,
+        "total": 3
+      },
+      "successful": 0,
+      "failed": 0,
+      "canceled": 0,
+      "timeout": 0,
+      "inactive_purged": 0,
+      "queues": []
+    }
```

Comment:
`flux job stats` doesn't display information about an individual job, it displays job statistics for the instance as a whole. This is the plumbing equivalent of `flux jobs --stats-only`, so maybe we should drop this section?

Reply:
okay I can drop! High level dinosaur note  here - as a user I'm always looking for a `flux job <id>` (something) that gives me a json dump of everything I know about a job. I know it's possible with the Python API but I haven't found a good equivalent here. Maybe just `flux job <jobid> --json` or something could do it?

Reply:
Good idea! I had thought we'd had an issue open on this already, but I cannot find it! The obvious choice might be `flux job info --all JOBID`, which would dump all known info "keys" in a single JSON object.